Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1244,7 +1244,23 @@ void TextEditor::insert_at_cursor_or_replace_selection(const StringView& text)
     VERIFY(is_editable());
     if (has_selection())
         delete_selection();
+
+    // check if adding a newline leaves the previous line as just whitespace
+    auto const clr_len = m_cursor.column();
+    auto const should_clear_last_ln = text == "\n"
```

Comment:
Please write out full words in variable names instead of using random abbreviations, unless the abbreviation makes it significantly clearer.